README.md

```diff
@@ -18,7 +18,7 @@ Recent developments and changes are tracked in our [changelog](https://github.co
 
 Sockeye requires:
 - **Python3**
-- [MXNet-0.12.0](https://github.com/dmlc/mxnet/tree/v0.12.0)
+- [MXNet-0.12.1](https://github.com/dmlc/mxnet/tree/v0.12.0)
```

Comment:
Minor versions appear to continue on the same branch, hence v0.12.0 here.

Reply:
But these links refer to git tags, correct? And there seems to be a tag for 0.12.1:
https://github.com/apache/incubator-mxnet/tree/0.12.1

Reply:
Good catch!  We were linking to the branch here but the tag is more correct.

Reply:
This links to the tag as far as I can tell :)